expr/functions/cairo/png/cairo.go

```diff
@@ -969,6 +970,13 @@ func marshalCairo(p PictureParams, results []*types.MetricData, backend cairoBac
 		yDivisors:   p.YDivisors,
 	}
 
+	if params.width >= 32567 {
```

Comment:
@deniszh Maybe, it would be nice to add a small comment about the origin of this magic number. If I saw this code without having current context, I'd be totally lost :)

Reply:
Good point, @grzkv !
I just added comment and foud out that I made a mistake, it's 32767 instead, and 32767  is also valid. Amended.